main.py

```diff
@@ -298,34 +259,61 @@ def status(message):
     LOGGER.info("Hostname: %s", hostname)
 
     # Start reading from sensors
-    sensor_thread = Thread(target=read_data,
-                           args=(output_sensors, input_sensors, queue))
+    sensor_thread = Thread(target=read_data, args=(output_sensors, input_sensors, queue))
     sensor_thread.start()
-
-    # Start server
-    LOGGER.info("Starting server")
-    server = CoAPServer(("224.0.1.187", 5683), multicast=True)
-    server.add_resource('data/', DataResource(queue, input_sensors))
-    server.add_resource('name={}'.format(hostname), DummyResource())
-
+    
+    #load config file
+    with open("config.yml", 'r') as ymlfile:
```

Comment:
The config file name is provided as a parameter to this function. Let's use that instead of hard coding it:

```
with open(config_file) as ymlfile:
```